app/uk/gov/hmrc/incometaxsubscriptioneligibility/services/ConvertConfigValuesService.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxsubscriptioneligibility.services
+
+import javax.inject.{Inject, Singleton}
+import uk.gov.hmrc.incometaxsubscriptioneligibility.config.AppConfig
+import uk.gov.hmrc.incometaxsubscriptioneligibility.models.controllist.ControlListParameter
+
+import scala.concurrent.ExecutionContext
+
+
+@Singleton
+class ConvertConfigValuesService @Inject()(appConfig: AppConfig)(
+  implicit ec: ExecutionContext
+) {
+
+  def convertConfigValues(): Set[ControlListParameter] =
+    ControlListParameter.getParameterMap.values.filterNot(appConfig.loadControlListConfig).toSet
+
```

Comment:
unit test/s?

Reply:
All the scenarios are tested in the IT tests. There is not much to be tested for this service.